common/src/main/scala/io/phdata/pulse/common/domain/LogEvent.scala

```diff
@@ -17,21 +17,23 @@
 package io.phdata.pulse.common.domain
 
 /**
- * Describes the expected LogEvent input Object
- *
- * @param category   Fully qualified Class
- * @param timestamp  Timestamp
- * @param level      Logging Level
- * @param message    Log message
- * @param threadName Thread name
- * @param throwable  Throwable message
- * @param properties MDC properties
- */
+  * Describes the expected LogEvent input Object
+  *
+  * @param category   Fully qualified Class
+  * @param timestamp  Timestamp
+  * @param level      Logging Level
+  * @param message    Log message
+  * @param threadName Thread name
+  * @param throwable  Throwable message
+  * @param hostname   Host Name
+  * @param properties MDC properties
+  */
 case class LogEvent(id: Option[String],
                     category: String,
                     timestamp: String,
                     level: String,
                     message: String,
                     threadName: String,
                     throwable: Option[String] = None,
+                    hostname: String,
```

Comment:
You'll have to make this value optional so we can keep backwards compatibility (like `throwable`)